wls-briefwahl-service/src/test/java/de/muenchen/oss/wahllokalsystem/briefwahlservice/configuration/nfcconverter/NfcConverterTest.java

```diff
@@ -0,0 +1,175 @@
+/*
+ * Copyright (c): it@M - Dienstleister für Informations- und Telekommunikationstechnik
+ * der Landeshauptstadt München, 2024
+ */
+package de.muenchen.oss.wahllokalsystem.briefwahlservice.configuration.nfcconverter;
+
+import static org.junit.jupiter.api.Assertions.assertArrayEquals;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
```

Comment:
Leerzeile darf raus

Reply:
fixed in 6a4b931891e6796ac1d600e5700c2c0ec24d10ec